packages/next/package.json

```diff
@@ -100,11 +100,13 @@
     ]
   },
   "dependencies": {
+    "@modelcontextprotocol/sdk": "1.15.1",
```

Comment:
Is this a problem? It doesn't seem great adding a dependency like this for an experimental-only feature.